scripts/module_load.sh

```diff
@@ -4,5 +4,7 @@
 module --verbose load gcc/12.2.0_gcc12.2.0
```

Comment:
do we still need module_load.sh if we build all in docker?

Reply:
no, will remove

Reply:
actually, I'll keep them until the sonar cloud job (failed and uses them) is move to docker